lib/iconv.js

```diff
@@ -96,6 +96,9 @@ function convert(input, context) {
   if (!(input instanceof Buffer) && input !== FLUSH) {
     throw new Error('Bad argument.');  // Not a buffer or a string.
   }
+  if (context && context.trailer && input === FLUSH) {
```

Comment:
Can you use strict equality like a few lines below?  Or maybe fold it into that if statement, I leave that up to you.